src/components/Item.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { View, Image, Text, ImageProps, StyleSheet } from "react-native";
+import Octicons from "@expo/vector-icons/Octicons";
+import { Typography } from "@/theme/typography";
+import { ColorTheme } from "@/theme/colors";
+
+export const Item = (props: ItemProps) => {
```

Comment:
I have found in the past any attempt to make Lists reusable other than just a parent wrapper is kind of pointless. Every time I run into a list it is artisanal every time so I would question the utility of making a reusable ListItem or List